test-files/cases/attribute/network/highway-2927-multiple-types-1/Expected.osm

```diff
@@ -883,9 +883,9 @@
         <tag k="z_order" v="-999999"/>
     </way>
     <way visible="true" id="-41" timestamp="2019-01-17T19:50:06Z" version="1">
-        <nd ref="-131"/>
-        <nd ref="-130"/>
         <nd ref="-129"/>
+        <nd ref="-130"/>
+        <nd ref="-131"/>
```

Comment:
It is interesting that a compiler change makes small changes to the code, enough to reverse this way and allow for two ways in the toy tests below to be joined that weren't being joined before.  Small subtle changes in the output for the unit tests.